naucse/templates/course.html

```diff
@@ -85,8 +56,88 @@
                     {% endfor %}
                 </div>
                 {% endfor %}
+
+            </div>
+            <div class="col-md-4">
+                {% if course.start_date %}
+                    <div class="card course-card text-center">
+                        <div class="card-header">
+                            Toto jsou podklady pro kurz s instruktorem
+                        </div>
+                        <div class="card-body">
+                            <div class="calendar-dates">
+                                {{ (course.start_date, course.end_date) | format_date_range }}
+                            </div>
+                        </div>
+                        <div class="card-footer small text-left">
+                            <ul class="list-unstyled" style="margin-bottom:0;">
+                                <li>
+                                    <a href="{{ url_for('course_calendar', course=course) }}">→ Kalendář</a>
+                                </li>
+                                {% if course.base_course %}
+                                <li>
+                                    <a href="{{ course_url(course.base_course) }}">→ Ekvivalentní kurz pro samouky</a>
+                                </li>
+                                {% endif %}
+                            </ul>
+                        </div>
+                    </div>
+                {% else %}
+                    <div class="card course-card">
+                        <div class="card-header text-center">
+                            Toto jsou materiály pro samouky
+                        </div>
+                        <div class="card-body">
+                            <p>
+                                Doufáme, že naše materiály jsou srozumitelné a přínosné.
+                                Pokud ne, ozvěte se prosím:
+                            </p>
+                            <ul>
+                                <li>
+                                    na <a href="https://www.facebook.com/groups/pyonieri/">Facebooku</a>,
```

Comment:
I don't like having (what I think of as) content in templates. However, for the lack of better possibilities, let's keep it that way for now.